demos/python_demos/bert_question_answering_demo/bert_question_answering_demo.py

```diff
@@ -0,0 +1,370 @@
+                t_end - t_start
+            ))
+
+            # get start-end scores for context
+            def get_score(name):
+                out = np.exp(res[name].reshape((max_length,)))
+                return out / out.sum(axis=-1)
+
+            score_s = get_score(output_names[0])
+            score_e = get_score(output_names[1])
+
+            # get 'no-answer' score (not valid if model has been fine-tuned on squad1.x)
+            if args.model_squad_ver.split('.')[0] == '1':
+                score_na = 0
+            else:
+                score_na = score_s[0] * score_e[0]
+
+            # find product of all start-end combinations to find the best one
+            c_s_idx = len(q_tokens_id) + 2  # index of first context token in tensor
+            c_e_idx = max_length - (1 + pad_len)  # index of last+1 context token in tensor
+            score_mat = np.matmul(
+                score_s[c_s_idx:c_e_idx].reshape((c_e - c_s, 1)),
+                score_e[c_s_idx:c_e_idx].reshape((1, c_e - c_s))
+            )
+            # reset candidates with end before start
+            score_mat = np.triu(score_mat)
+            # reset long candidates (>max_answer_token_num)
+            score_mat = np.tril(score_mat, args.max_answer_token_num - 1)
+            # find the best start-end pair
+            max_s, max_e = divmod(score_mat.flatten().argmax(), score_mat.shape[1])
+            max_score = score_mat[max_s, max_e] * (1 - score_na)
+
+            # convert to context text start-end index
+            max_s = c_tokens_se[c_s + max_s][0]
+            max_e = c_tokens_se[c_s + max_e][1]
+
+            # check that answers list does not have duplicates (because of context windows overlapping)
+            same = [i for i, a in enumerate(answers) if a[1] == max_s and a[2] == max_e]
+            if same:
+                assert len(same) == 1
+                # update existing answer record
+                a = answers[same[0]]
+                answers[same[0]] = (max(max_score, a[0]), max_s, max_e)
+            else:
+                # add new record
+                answers.append((max_score, max_s, max_e))
+
+            # check that context window reached the end
+            if c_e == len(c_tokens_id):
+                break
+
+            # move to next window position
+            c_s = min(c_s + c_stride, len(c_tokens_id))
+            c_e = min(c_s + c_wnd_len, len(c_tokens_id))
+
+        t1 = time.perf_counter()
+        log.info("The performance below is reported only for reference purposes, "
+                 "please use the benchmark_app tool (part of the OpenVINO samples) for any actual measurements.")
+        log.info("{} requests of {} length were processed in {:0.2f}sec ({:0.2}sec per request)".format(
+            t_count,
+            max_length,
+            t1 - t0,
+            (t1 - t0) / t_count
+        ))
+
+        # print top 3 results
+        answers = list(sorted(answers, key=lambda x: -x[0]))
```

Comment:
```suggestion
        answers = sorted(answers, key=lambda x: -x[0])
```

`sorted` already returns a list.